File: packages/nx-micronaut-maven/src/generators/library/files/java/src/test/java/__packageDirectory__/HelloServiceTest.java
package <%= packageName %>;

import jakarta.inject.Inject;

import org.junit.jupiter.api.Assertions;
import org.junit.jupiter.api.Test;

import io.micronaut.test.extensions.junit5.annotation.MicronautTest;

@MicronautTest
public class HelloServiceTest {

    @Inject
    HelloService service;

    @Test
    public void testGreetingService() {
        Assertions.assertEquals("Hello World", service.greeting());
    }
}

File: packages/internal/generators-files/micronaut/library/shared/kotlin/src/test/kotlin/__packageDirectory__/HelloServiceTest.kt
package <%= packageName %>

import org.junit.jupiter.api.Assertions;
import org.junit.jupiter.api.Test
import io.micronaut.test.extensions.junit5.annotation.MicronautTest
import jakarta.inject.Inject


@MicronautTest
class HelloServiceTest {


  @Inject
  lateinit var service: HelloService


  @Test
  fun `Should return Hello World`() {
    println(">> Should return Hello World")
    Assertions.assertTrue(service.greeting().contains("Hello World"));
  }


}

